Dockerfile.dev

```diff
@@ -5,5 +5,8 @@ RUN apk update && apk add postgresql-dev gcc python3-dev musl-dev bash git libff
 
 WORKDIR /usr/src/app
 
+COPY requirements.txt .
+COPY setup.py .
```

Comment:
👍 - I removed these because I assumed that docker-compose would make the code available (through volumes), but apparently I didn't test it properly. Thanks!